### PR TITLE
Add a mutex lock to allow only one instance of mimic at a time

### PR DESCRIPTION
**Pull request by UnknwonWarlock**
# Changes
- Add a mutex lock to allow only one instance of Mimic to run at a time
- Add pywin32 dependency to Pipfile

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -106,5 +110,8 @@ def on_gui_quit():
 
 
 if __name__ == "__main__":
-    multiprocessing.freeze_support()
-    main()
+    handle = CreateMutex(None, 1, "Mimic Mutex")
+    
+    if GetLastError() != ERROR_ALREADY_EXISTS:
+        multiprocessing.freeze_support()
```
Comment (link00000000) [inline main.py:116]: So I don't understand why, but if this line isn't directly after `if __name__ == "__main__"`, the subprocess that runs the webserver will not be started.

**Review by link00000000 — CHANGES_REQUESTED** [review]
Webserver does not start anymore

File: main.py (hunk 1 above)
Comment (UnknwonWarlock) [reply main.py]: Can you check if my commit that just moves it fixes the issue?

Comment (link00000000) [reply main.py]: Seems to be working now, not sure why it needs to be called so early on. 👍🏻

**Comment by link00000000** [comment]
Good find on the windows mutexes

**Pull request merged**